examples/applications/plot_stock_market.py

```diff
@@ -19,8 +19,8 @@
 #
 # The data is from 2003 - 2008. This is reasonably calm: (not too long ago so
 # that we get high-tech firms, and before the 2008 crash). This kind of
-# historical data can be obtained from APIs like the quandl.com and
-# alphavantage.co .
+# historical data can be obtained from APIs like the `data.nasdaq.com <https://data.nasdaq.com/>`_ and
+# `alphavantage.co <https://www.alphavantage.co/>`_.
```

Comment:
Last nit: May we wrap this line so it is more consistent with the rest of the file?

```suggestion
# historical data can be obtained from APIs like the
# `data.nasdaq.com <https://data.nasdaq.com/>`_ and
# `alphavantage.co <https://www.alphavantage.co/>`_.
```